english/meditate/meditate.md

```diff
@@ -1,6 +1,8 @@
 You recently read that meditation helps you feel more awake, relieves stress, and helps you prepare for the day ahead.
 
-So you begin to meditate for 30 minutes and when you finish, you feel more refreshed than ever.
+You decide to whip out your iPhone 4s (yes, it's probably about time that you upgrade your phone) and open the Headspace meditation app.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.